src/test/java/bio/terra/janitor/integration/common/configuration/TestConfiguration.java

```diff
@@ -10,9 +15,28 @@
 @EnableConfigurationProperties
 @ConfigurationProperties(prefix = "test")
 public class TestConfiguration {
+  /** How long to keep the resource before the 'prod' Janitor do the cleanup. */
+  public static Duration RESOURCE_TIME_TO_LIVE = Duration.ofMinutes(30);
+
+  /** pubsub project id to publish track resource to Janitor test env(toolsalpha) */
+  private String resourceProjectId;
+
+  /** pubsub project id to publish track resource to Janitor prod env(tools) */
+  private String prodTrackResourceProjectId;
+
+  /** pubsub topic id to publish track resource to Janitor test env(toolsalpha) */
   private String trackResourceTopicId;
 
-  private String clientServiceAccountPath;
+  /** pubsub topic id to publish track resource to Janitor prod env(tools) */
+  private String prodTrackResourceTopicId;
+
+  /** Credential file path to be able to pubish message to Janitor test env(toolsalpha). */
+  private String janitorClientServiceAccountPath;
```

Comment:
wdyt about having a nested class for the pubsub project id, topic id, and service account to use for publishing? I think those three fields ar etied together and it'd be clearer to have them grouped  with an instance for test env and prod env.
```
public class TrackPubSubConfiguration {
  private String projectId;
  private String topicId;
  private String clientServiceAccountPath;

  public GoogleCredentials getClientCredentialsOrDie() { ... }
}
```